### DISCUSSION TITLE
How to view eslint warning messages?

### DISCUSSION THREAD

**Discussion by mksonkar** [LazyVim/LazyVim#4714] [Q&A] [answered]
In the image below, the eslint warning is cut-off from the screen. How can I view the complete message?
![image](https://github.com/user-attachments/assets/f2bfde22-9a83-4a4e-aac4-068bb7fdaf86)

**Comment by dpetka2001** [comment]
`<leader>cd`

**Reply by mksonkar** [reply]
thanks for your answer, but in my case its ```<leader>xx``` to view Trouble diagnostics, and the warnings were there

**Comment by mksonkar** [comment, marked as answer]
Sorry, I just found out how, ```<Leader>xx```
![image](https://github.com/user-attachments/assets/af2f1bcd-a224-41d8-bfbe-037b1fa2ebf9)
